src/streamlink/compat.py

```diff
@@ -26,11 +26,17 @@ def devnull():
     def bytes(b, enc="ascii"):
         return _str(b)
 
+    try:
+        from cStringIO import StringIO
```

Comment:
This is more like `BytesIO` than `StringIO`, it doesn't support unicode.

Reply:
It supports unicode, but only in one direction.

Do you need a revert of this to remove all compat changes?